index.ts

```diff
@@ -288,29 +290,39 @@ function getProjectSettings(failSilently?: boolean): Promise<ProjectSettings> {
  * Required before every API call.
  * @param {Function} cb The callback
  */
-function getAPICredentials(cb: (rc: ClaspSettings | void) => void) {
-  DOTFILE.RC.read().then((rc: ClaspSettings) => {
-    oauth2Client.setCredentials(rc);
-    cb(rc);
-  }).catch((err: object) => {
-    process.exit(-1);
-  });
+function getAPICredentials(isLocal: boolean, cb: (rc: ClaspSettings | void) => void) {
+  if (isLocal) {
+    DOTFILE.RC_LOCAL.read().then((rc: ClaspSettings) => {
+      oauth2Client.setCredentials(rc);
+      cb(rc);
+    }).catch((err: object) => {
+      process.exit(-1);
+    });
+  } else {
+    DOTFILE.RC.read().then((rc: ClaspSettings) => {
+      oauth2Client.setCredentials(rc);
+      cb(rc);
+    }).catch((err: object) => {
+      process.exit(-1);
+    });
+  }
 }
 
 /**
  * Requests authorization to manage Apps Script projects.
  * @param {boolean} useLocalhost True if a local HTTP server should be run
  *     to handle the auth response. False if manual entry used.
  */
-function authorize(useLocalhost: boolean) {
+function authorize(useLocalhost: boolean, writeToLocalKey: boolean) {
```

Comment:
For every `local`, say `own`.
`writeToOwnKey`.

The global key is also local.